src/app/(guest)/login/page.tsx

```diff
@@ -40,19 +46,27 @@ const Page = () => {
       </div>
       <div className="flex flex-1 flex-col justify-center bg-slate-100 px-20">
         <h3 className="mb-6 text-2xl">Sign in</h3>
-        <Input
-          name="email"
-          type="email"
-          label="Email"
-          placeholder="Enter your email"
-        />
-        <Input
-          name="password"
-          type="password"
-          label="Password"
-          placeholder="Enter your password"
-        />
-        <Button onClick={submitForm}>Sign in</Button>
+        <form onSubmit={formik.handleSubmit}>
+          <Input
+            name="email"
+            type="email"
```

Comment:
is there a frontend validation happening if I type an invalid email? or does it still allow me to submit and the API request happens?

Reply:
The input field type `email` and also we are using `yup` package for validation. And if you enter invalid email then it will show `Invalid email format` error and it's not allow to submit the API request.